addons/sourcemod/scripting/VScript_ze_kitchen.sp

```diff
@@ -252,12 +252,35 @@ public void OnChangeEggsCount(const char[] output, int caller, int activator, fl
 		g_Fly.IncrementEggCount(-1);
 }
 
-public void OnEntitySpawned(int entity, const char[] classname)
```

Comment:
you completely remove the function, do like here:

- https://github.com/srcdslab/sm-plugin-BossHP/blob/c14b628527d0070721554dee53a3fee74bf32b35/addons/sourcemod/scripting/BossHP.sp#L126C1-L129C2